client/src/components/Lesson/WordCreator/WordRootSelector.tsx

```diff
@@ -54,35 +52,30 @@ const WordRootSelector = (props: WordRootSelectorProps) => {
             props.onChange(newRoots);
         };
 
-        const deleteIcon = props.disabled ? undefined :
-            <IconButton iconClassName='material-icons'
-                        tooltip='Remove'
-                        iconStyle={{color: '#AAA'}}
-                        onClick={() => props.onChange(props.roots.filter((_, i) => i !== index))}>
-             clear
-            </IconButton>;
-
         return (<div key={root.root} style={{ display: 'flex' }}>
             <TextField
                 name={root.root}
                 value={root.root}
                 hintText='Root'
                 style={{ width: '20%'}}
-                onChange={onRootChange}
-                disabled={props.disabled} />
+                onChange={onRootChange} />
             <TextField
                 name={root.meaning}
                 value={root.meaning}
                 hintText='Root meaning'
                 style={{ width: '75%', marginLeft: '5%' }}
-                onChange={onMeaningChange}
-                disabled={props.disabled} />
```

Comment:
Editing+locked is now the study view so we'll have 2 states: editing and study view